src/adapter/vertex.rs

```diff
@@ -36,14 +36,16 @@ pub enum VertexKind<'a> {
     ImplementedTrait(&'a Path, &'a Item),
     FunctionParameter(&'a str),
     FunctionAbi(&'a Abi),
+    Discriminant(Cow<'a, str>),
+    Variant(EnumVariant<'a>),
```

Comment:
`EnumVariant` might have to be `pub`. Let's keep it in sync visibility-wise with other types exposed in variants here, like `ImportablePath` and `AttributeMetaItem`.